osu.Game/Skinning/Editor/SkinBlueprint.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Screens.Play.HUD;
+using osuTK;
+
+namespace osu.Game.Skinning.Editor
+{
+    public class SkinBlueprint : SelectionBlueprint<ISkinnableComponent>
+    {
+        /// <summary>
+        /// The <see cref="DrawableHitObject"/> which this <see cref="OverlaySelectionBlueprint"/> applies to.
+        /// </summary>
+        public readonly ISkinnableComponent Component;
```

Comment:
Sorta redundant. `SelectionBlueprint<T>.Item` exists in base. Could either delegate to it or just use that directly.

Reply:
Also wrong xmldoc.